subquery/network-contracts
# add config for kepler launch

## Changes:

- Apply latest Kepler configuration
- Add `totalSupply` as a param when deploy `SQToken` contract
- Add totalSupply in verify script
